x-pack/plugins/spaces/public/nav_control/nav_control.tsx

```diff
@@ -40,7 +40,15 @@ export function initSpacesNavControl(
 
       ReactDOM.render(
         <KibanaRenderContextProvider {...core}>
-          <Suspense fallback={<EuiLoadingSpinner />}>
+          <Suspense
+            fallback={
+              <EuiSkeletonRectangle
+                css={{ marginInline: 8 }}
```

Comment:
We could pull in `euiThemeVars` and set this to `${euiThemeVars.euiSizeS}`, but it feels like potential overkill for this single style override for this file. Happy to do it, if preferable.

Reply:
Without this, the button/loader jumps right.